src/writer/audit_and_fix.py

```diff
@@ -0,0 +1,46 @@
+"""
+This module allows you to migrate obsolete data structures to newer data structures.
+For example, when loading the ui.
+"""
+
+
+def fix_components(components: dict) -> dict:
```

Comment:
@ramedina86 Since the implementation of pydantic, the structure of the ui.json file has been evaluated on the backend side. I had to implement the fix processing when loading the file before it was integrated into pydantic.

Reply:
That's a good point...